src/python/Dockerfile

```diff
@@ -110,6 +110,33 @@ COPY --chown=grapl --from=python-test-deps /home/grapl/python_test_deps python_t
 RUN python_test_deps/install_requirements.sh
 
 
+# grapl-http-service
+################################################################################
+
+# build
+FROM grapl-python-build AS grapl-http-service-build
+
+COPY --chown=grapl python/grapl-http-service grapl-http-service
+COPY --chown=grapl --from=grapl-common-build /home/grapl/grapl-common grapl-common
+
+RUN source venv/bin/activate && \
+    cd grapl-common && \
+    pip install . && \
+    python setup.py sdist bdist_wheel
+
+RUN source venv/bin/activate && \
+    cd grapl-http-service && \
+    pip install . && \
+    python setup.py sdist bdist_wheel
```

Comment:
We should ultimately be using the wheel built by Pants, but haven't established this pattern yet. I'll work on that today.